assets/sass/patterns/atoms/buttons.scss

```diff
@@ -26,6 +26,10 @@
   text-transform: uppercase;
 }
 
+.button--fixed-width {
```

Comment:
This class name should change because it's for a specific use case.

Reply:
Moved to modal-window css file and renamed .modal-content__clipboard-btn